cloud

```diff
@@ -1234,6 +1234,7 @@ function cloud_function_withdraw() {
 # Output deployment environment for Cloud Functions
 # Args: --format=yaml|sh
 #       --log-level=NAME
+#       --optimize=INT
```

Comment:
This only adds the documentation. You gotta make the function actually accept the argument, and then add the actual `PYTHONOPTIMIZE` variable to the environment. See how it's done for the `--log-level` option, which is stored in the `log_level` variable inside the function.

Reply:
check now